packages/commonjs/test/helpers/util.js

```diff
@@ -9,7 +9,7 @@ function commonjs(options) {
 
 function requireWithContext(code, context) {
   const module = { exports: {} };
-  const contextWithExports = Object.assign({}, context, { module, exports: module.exports });
+  const contextWithExports = { ...context, module, exports: module.exports };
```

Comment:
Nice, it was not aware Node 10 already supported this but apparently it does!

Reply:
My eyes could not take this anymore, so I had to tweak this